secant/Dependencies/WalletConfigProvider/FeatureFlagsManagerInterface.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  FeatureFlagsManagerInterface.swift
+//  secant
+//
+//  Created by Michal Fousek on 23.02.2023.
+//
+
+import ComposableArchitecture
+
+extension DependencyValues {
+    var featureFlagsManager: FeatureFlagsManagerClient {
```

Comment:
I'd change the name of this for something configuration related

Reply:
This is just rebasing error. This file exists only on disk and isn't even used in the project.